blackbox/docs/sql/statements/alter-cluster.rst

```diff
@@ -50,3 +51,21 @@ get acknowledged.
     Additionally, keep in mind that this statement only triggers the shard
     re-allocation and is therefore asynchronous. Unassigned shards with large
     size will take some time to allocate.
+
+``SET LICENSE``
+---------------
+
+Registers license information to the cluster state based on the ``license_parameters``::
```

Comment:
`to the cluster state` is imho unnecessary. It's an implementation detail where it is stored or kept. We should focus this in behavior relevant for the user. 

E.g. mention the validation and what happens if it is rejected?